sdk/src/manifest.rs

```diff
@@ -1158,4 +1158,30 @@ pub(crate) mod tests {
             TEST_SMALL_JPEG
         );
     }
+
+    #[cfg(all(feature = "file_io", feature = "xmp_write"))]
+    #[actix::test]
+    async fn test_embed_sidecar_with_parent_manifest() {
```

Comment:
Why is this an `async` test? I don't see any `await` or `async` inside the function.

Reply:
good point, I removed async from some of the other tests that didn't need it.